.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @smukherj1 @davegay @eytankidron @alexeagle @pcj @gravypod
```

Comment:
I suggest removing @davegay and @eytankidron here as they're no longer actively involved with rules_docker

Reply:
cool thx